eslint/babel-eslint-shared-fixtures/config/babel.config.js

```diff
@@ -7,17 +7,10 @@ module.exports = {
     "@babel/preset-react",
   ],
   plugins: [
-    "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-syntax-import-meta",
     "@babel/plugin-syntax-export-default-from",
     "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-proposal-nullish-coalescing-operator",
-    "@babel/plugin-proposal-optional-chaining",
-    "@babel/plugin-syntax-numeric-separator",
-    "@babel/plugin-syntax-export-namespace-from",
     ["@babel/plugin-proposal-decorators", { decoratorsBeforeExport: false }],
     ["@babel/plugin-proposal-pipeline-operator", { proposal: "minimal" }],
-    "@babel/plugin-syntax-bigint",
```

Comment:
They are removed because one of the following applies:
- It is enabled by default in latest `@babel/parser`
- It is included in `@babel/preset-env`